dashboard/test/controllers/potential_teachers_controller_test.rb

```diff
@@ -7,20 +7,22 @@ class PotentialTeachersControllerTest < ActionDispatch::IntegrationTest
       post potential_teachers_url, params: {
         name: 'foosbars',
         email: 'foobar@example.com',
-        source_course_offering_id: course_offering.id
+        source_course_offering_id: course_offering.id,
+        receives_marketing: true
       }
     end
   end
 
   test "show returns correct information for a potential teacher" do
     course_offering = create :course_offering, key: 'course-offering-1', display_name: "Test"
-    example_potential_teacher = create :potential_teacher, source_course_offering_id: course_offering.id
+    example_potential_teacher = create :potential_teacher, source_course_offering_id: course_offering.id, receives_marketing: true
```

Comment:
I think `receives_marketing: true` isn't necessary here since you have that set in the factory file!

Reply:
🤦🏻‍♀️ You right you right! Thanks!